examples/seismic/tti/tti_example.py

```diff
@@ -37,12 +37,23 @@ def run(shape=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
     # Define receiver geometry (spread across `x, y`` just below surface)
     rec, u, v, summary = solver.forward(save=save, autotune=autotune)
 
-    if preset == 'constant':
-        # With  a new m as Constant
-        v0 = Constant(name="v", value=2.0, dtype=np.float32)
-        solver.forward(save=save, vp=v0)
-        # With a new vp as a scalar value
-        solver.forward(save=save, vp=2.0)
+    if preset in ['constant-tti', 'constant-tti-noazimuth']:
```

Comment:
@ofmla wait... so this way it will run twice if preset=constant-tti for example ? right before this `if` and then inside?